dev/archery/archery/cli.py

```diff
@@ -789,7 +789,9 @@ def integration(with_all=False, random_seed=12345, **args):
               default='-', required=True)
 @click.option('--arrow-token', envvar='ARROW_GITHUB_TOKEN',
               help='OAuth token for responding comment in the arrow repo')
-def trigger_bot(event_name, event_payload, arrow_token):
+@click.option('--crossbow-token', '-ct', envvar='CROSSBOW_GITHUB_TOKEN',
+              help='OAuth token for pushing to the crossow repository')
+def trigger_bot(event_name, event_payload, arrow_token, crossbow_token):
```

Comment:
We don't need to revert this change because `crossbow_token` isn't used in this function.